presentation/src/main/java/com/mashup/presentation/login/LoginScreen.kt

```diff
@@ -1,57 +1,79 @@
 package com.mashup.presentation.login
 
+import androidx.compose.foundation.ExperimentalFoundationApi
 import androidx.compose.foundation.Image
 import androidx.compose.foundation.clickable
 import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
 import androidx.compose.material.Scaffold
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
 import com.mashup.presentation.R
+import com.mashup.presentation.ui.common.KeyLinkButton
 import com.mashup.presentation.ui.common.KeyLinkMintText
 
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
-fun LoginScreen(
+fun LoginScreen(viewModel: LoginViewModel = hiltViewModel()) {
```

Comment:
```suggestion
fun LoginScreen(viewModel: LoginViewModel = viewModel()) {
```
이 방식도 괜찮을 것 같아용~
우리가 사실 Compose의 Navigation을 사용하는게 아니라 굳이 hiltViewModel을 사용하지 않아도 괜찮을 것 같습니당~
https://velog.io/@wlsrhkd4023/Compose-hiltViewModel%EA%B3%BC-viewModel-%EC%B0%A8%EC%9D%B4

Reply:
수정 완 ~! 공유한 페이지 더 읽어보고 공부해야겠다 곰마워